pkg/account/service.go

```diff
@@ -215,6 +218,26 @@ func (a *Service) Delete(data *Account) error {
 		return err
 	}
 
+	err = a.managerSvc.DeletePrincipalAccess(data)
+	if err != nil {
+		return err
+	}
+
+	err = a.dataSvc.Delete(data)
```

Comment:
Looks like this is a duplicate call, see line 216

Reply:
Yikes good catch.